shared/fs/header/header.js

```diff
@@ -80,6 +81,7 @@ const FolderHeader = ({
       )}
     </Box>
     <ConnectedFilesBanner path={path} />
+    <ConnectedResetBanner path={path} />
```

Comment:
Have you tried having them together and see if they'd become too big? If so perhaps we can hide the FilesBanner if ResetBanner needs to show.

Reply:
Also good call.